### PR TITLE
Feat/improved tests

### PR DESCRIPTION
**Pull request by sakulstra**
- added tests
- removed the standalone oracle, as it's no longer needed

### CODE REVIEW

File: tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+    contracts.emissionManager.setEmissionAdmin(rewardToken, emissionAdmin);
+  }
+
+  function test_2701() external view {
+    assertEq(
+      keccak256(abi.encode(uint256(keccak256('aave-dao.storage.ERC20AaveLM')) - 1)) &
+        ~bytes32(uint256(0xff)),
+      0x4fad66563f105be0bff96185c9058c4934b504d3ba15ca31e86294f0b01fd200
+    );
+  }
```
Comment (kyzia551) [inline tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol:94]: So, without making the *StorageLocation we can't test it properly...

File: tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+  ) public {
+    TestEnv memory env = _setupTestEnvironment(
+      depositAmount,
+      emissionEnd,
+      emissionPerSecond,
+      waitDuration
+    );
+
+    uint256 claimable = lmUpgradeable.getClaimableRewards(user, rewardToken);
+    assertLe(claimable, env.emissionDuration * env.emissionPerSecond);
```
Comment (kyzia551) [inline tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol:124]: so, by this we check, that we will not overpay. But can we have some lower bound?

Comment (sakulstra) [reply tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol:124]: imo this should not be on stata tests. If it all should be on rewardscontroller itself.

File: tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol (hunk 1 above)
Comment (sakulstra) [reply tests/periphery/static-a-token/ERC20AaveLMUpgradable.t.sol:94]: yes, as long as it's private there's no reasonable way to test i think

**Pull request merged**